src/_data/sectionApp.json

```diff
@@ -0,0 +1,36 @@
+{
+  "ru": {
+      "sectionTitle": "как работает приложение",
+      "items": [
```

Comment:
I'd suggest keeping images names and alt for them also in data. It'll be more universal and will help with different images for different languages. Activating by id, but src keep in data separately